pylintrc

```diff
@@ -53,7 +53,8 @@ disable=
     too-many-return-statements,
     too-many-branches,
     too-many-arguments,
-    too-many-locals
+    too-many-locals,
+    missing-docstring
```

Comment:
@nedbat Because the quality build was failing from too many missing docstrings (and no other issues) and we decided to remove the caveat that make quality could still pass with 20 pylint errors. Seemed better to ignore the one issue that was still happening rather than let a build succeed because there are < 20 errors.